docs/reference/settings.md

```diff
@@ -43,3 +43,4 @@ Along with the common settings, the following driver-specific settings are avail
 | `webScreenshotMode` | `native` or `page` or `viewport` | Defines the screenshoting logic if the current context is set to a web one. The default value is `native`, which makes the driver to take screenshots from WDA, e.g. the whole device screen including status bars. The `page` mode tries to retrieve the screenshot of the whole active web page, while the `viewport` one only retrieves a shot of the visible viewport. |
 | `useClearTextShortcut` | `boolean` | Whether to use the fastest operation (using IOHIDEvent) to clear texts. In headless mode, simulator's keyboard won't show up anymore after clearing texts using this approach in some cases (see [this issue](https://github.com/appium/appium/issues/20727) for more details). Defaults to true |
 | `limitXPathContextScope` | `boolean` | Due to historical reasons XCUITest driver limits scopes of element context-based searches to the parent element. This means a request like `findElement(By.xpath, "//XCUIElementTypeButton").findElement(By.xpath, "./..")` would always fail, because the driver only collects descendants of the `XCUIElementTypeButton` element for the destination XML source. The `limitXPathContextScope` setting being set to `false` changes that default behavior, so the collected page source includes the whole page source XML where `XCUIElementTypeButton` node is set as the search context. With that setting disabled the search query above should not fail anymore. Although, you must be careful while building XPath requests for context-based searches with the `limitXPathContextScope` setting being set to `false`. A request like `findElement(By.xpath, "//XCUIElementTypeAlert").findElement(By.xpath, "//XCUIElementTypeButton")` would ignore the current context and search for `XCUIElementTypeButton` through the whole page source. Use `.` notation to correct that behavior and only find `XCUIElementTypeButton` nodes which are descendants of the `XCUIElementTypeAlert` node: `findElement(By.xpath, "//XCUIElementTypeAlert").findElement(By.xpath, ".//XCUIElementTypeButton")`. |
+| `skipDocumentsContainerCheckInFileManagement` | `boolean` | If skip checking documents container type in file management for real devices. `false` by default.|
```

Comment:
skipDocumentsContainerCheckInFileManagement -> skipDocumentsContainerCheck

Reply:
If skip checking documents container type in file management for real devices. -> Whether to apply special handling for the `documents` container type while pushing files to real devices (`false`, the default value) or treat it in the same way as any other container type (`true`).

Reply:
Updated.

I initially used `skipDocumentsContainerCheck` but added `InFileManagement` to explain this is file management related to give more info only from the naming, while it is actually longer naming though. Just a note.